pkg/topology/query.go

```diff
@@ -2,18 +2,25 @@ package topology
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"net/url"
 	"strconv"
 	"strings"
 
+	jsontime "github.com/liamylian/jsontime/v2/v2"
+	"github.com/pkg/errors"
+
 	"github.com/flanksource/canary-checker/pkg"
 	"github.com/flanksource/canary-checker/pkg/db"
 	"github.com/flanksource/commons/logger"
-	"github.com/pkg/errors"
 )
 
+var json = jsontime.ConfigWithCustomTimeFormat
+
+func init() {
+	jsontime.AddTimeFormatAlias("postgres_timestamp", "2006-01-02T15:04:05.999999999")
```

Comment:
Isn't this duplicated ?

Reply:
They are initialized in two different packages... one is the `topology` package and another one is the `pkg` package